contracts/contracts/rem.system/src/voting.cpp

```diff
@@ -178,7 +178,6 @@ namespace eosiosystem {
        * their first vote and should consider their stake activated.
        */
       if( voter->last_vote_weight <= 0.0 ) {
-         _gstate.total_activated_stake += voter->staked;
```

Comment:
this is still needed for new chains to activate correctly.

Reply:
@roman-tik, done.